build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/MrjarPlugin.java

```diff
@@ -62,13 +62,13 @@ public class MrjarPlugin implements Plugin<Project> {
     public void apply(Project project) {
         project.getPluginManager().apply(ElasticsearchJavaBasePlugin.class);
         var javaExtension = project.getExtensions().getByType(JavaPluginExtension.class);
-        var isIdea = System.getProperty("idea.active", "false").equals("true");
```

Comment:
This is active all the time when triggering gradle from within idea and not only limited to importing project action